CONTRIBUTING.md

```diff
@@ -202,12 +202,6 @@ notes will look like for the current state of the repo you can run:
 and the release notes in particular will be located at
 `docs/_build/html/release_notes.html`
 
-## Installing Qiskit IBM Provider from source
```

Comment:
These docs don't exist anymore. Since this repo is going away, probably not worth adding a replacement.